WooCommerce/Classes/POS/Presentation/POSFloatingControlView.swift

```diff
@@ -9,6 +9,7 @@ struct POSFloatingControlView: View {
     @Binding private var showSupport: Bool
     @Binding private var showDocumentation: Bool
     @State private var showProductRestrictionsModal: Bool = false
+    @State private var showBarcodeScanning: Bool = false
```

Comment:
```suggestion
    @State private var showBarcodeScanningInformation: Bool = false
```
Nit: with plans to add camera-based scanning, which will also have a view to show, this might be a bit clearer in the future.